.github/workflows/wheel_noarch.yml

```diff
@@ -56,7 +56,7 @@ jobs:
   upload-pypi:
     needs: build-pure-python-wheel
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master'}}
+    if: ${{ github.event_name == 'release' }}
```

Comment:
To be sure, so here we do not want to upload to PyPI unless we're doing a release, right?

Reply:
This is for TestPyPI, and yep, that is correct.
We always manually upload wheel to PyPI to avoid any fallout of mistakes.